src/index.js

```diff
@@ -0,0 +1,137 @@
+
+const DEFAULT_CITY = 'Seattle';
+// Select the HTML Element the event will occur on
+const increaseTempElement = document.getElementById('increaseTempBtn');
+const decreaseTempElement = document.getElementById('decreaseTempBtn');
+const currentTempTag = document.getElementById('currentTemp');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const cityNameReset = document.getElementById('cityNameReset');
+const SkyDropDown = document.getElementById('SkyChoice');
+const realtimeTemp = document.getElementById('realTempBtn');
+
+const changingTempColorAndLandscape = () => {
+  let currentTemp = parseInt(currentTempTag.textContent);
+  if (currentTemp >= 80) {
+    currentTempTag.className = 'red';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (currentTemp >= 70) {
+    currentTempTag.className = 'orange';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (currentTemp >= 60) {
+    currentTempTag.className = 'yellow';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (currentTemp >= 50) {
+    currentTempTag.className = 'green';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    currentTempTag.className = 'teal';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const changingSky = () => {
+  if (SkyDropDown.value === 'sunny') {
+    sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (SkyDropDown.value === 'cloudy') {
+    sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (SkyDropDown.value === 'rainy') {
+    sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else {
+    sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+
+//Wave 4 API
+
+
+const findLatLon = () => {
+  const query_city = cityNameInput.value;
+  console.log(query_city);
+  return axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+      q: query_city,
+      }
+    })
+    .then((response) => {
+      console.log(response);
+      const latitude = response.data[0].lat;
+      const longitude = response.data[0].lon;
+      console.log('success!', latitude, longitude);
+      return { lat: latitude, lon: longitude };
+    })
+    .catch((error) => {
+      console.log('findLatLon error!');
+    });
+};
+
+const getWeather = (query) => {
+  // console.log(query.lat)
+  return axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: query.lat,
+        lon: query.lon,
+      },
+    })
+    .then((response) => {
+      console.log(response.data.main.temp);
+      const temp = Math.floor((response.data.main.temp - 273.15) * 1.8 + 32);
+      return temp;
+    })
+    .catch((error) => {
+      console.log('getWeather error!');
+      console.log(error);
+    });
+};
+const connectFunctions = () => {
+  return findLatLon()
+  .then((latlon) => getWeather(latlon))
+}
+realtimeTemp.addEventListener('click', () => {
```

Comment:
Consider wrapping all the code where you use addEventListener inside a single method, something like registerEventHandlers() to keep the code in this file organized